website/addons/figshare/settings/defaults.py

```diff
@@ -1,7 +1,3 @@
-# FigShare application credentials
-CLIENT_ID = None
```

Comment:
To be consistent with the other addons and to make the tests pass, let's leave this in.